Bootstraps/AsyncInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace PHPPM\Bootstraps;
+
+
+use React\EventLoop\LoopInterface;
+
+interface AsyncInterface
```

Comment:
if this PR doesnt reference it, drop it from the PR and add it with the some you will do in the future.. makes sense?